recipes/brigand/all/conanfile.py

```diff
@@ -13,29 +14,51 @@ class BrigandConan(ConanFile):
     topics = ("meta-programming", "boost", "runtime", "header-only")
     homepage = "https://github.com/edouarda/brigand"
     license = "BSL-1.0"
-    settings = "compiler"
-    requires = "boost/1.75.0"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "with_boost": [True, False],
+    }
+    default_options = {
+        "with_boost": True,
+    }
+
     no_copy_sources = True
```

Comment:
Is this correct. In all other conanfiles this is called `no_copy_source`?

Reply:
Indeed, it was incorrect from the beginning